sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/common/cloud-provider/fix.adoc

```diff
@@ -0,0 +1,73 @@
+== How to fix it
+
+To eliminate the security risks associated with leaked cloud provider secrets,
+it is important to list where this secret is used, who or what is using it, and
+carefully proceed with the following steps.
+
+=== Revoke the secret
+Once a secret access key is compromised or leaked, you must immediately revoke
+it, change it, and apply the changes to the entity using it.
+
+It is also important that the number of entities using this secret is as close
+to **1** as possible. If revoking and changing a secret results in multiple
+denials of service, this is another problem: passwords, secrets, and any type
+of credentials should only be used to authenticate a single entity (a person or
+a system).
+
+Revocation can be done via the cloud provider's web application or via its CLI
+and APIS.
+
+By revoking the secret access key, the compromised credentials are no longer
```

Comment:
> By revoking the secret access key, the compromised credentials are no longer valid, which immediately stops the movements of a particular intruder.

I don't think that is true. The attacker could have used the credentials to add a backdoor. Then they can continue even though the credentials are not valid anymore.